candi/tools/functions.sh

```diff
@@ -18,21 +18,28 @@ set -Eeo pipefail
 
 function check_jq() {
     if ! jq --version &>/dev/null; then
-      >&2 echo "ERROR: jq is not installed"
+      >&2 echo "ERROR: jq is not installed. Please install it from https://stedolan.github.io/jq/download"
+      return 1
+    fi
+}
+
+function check_gh() {
+    if ! gh --version &>/dev/null; then
```

Comment:
Should I be authorized to use the gh utility? Where should I get the token? I think it is obvious for some engineers, but for others, it may not.

Reply:
We can avoid manual dependencies management if we use GA schedule right away. We could run it every day. 

See 
https://github.com/deckhouse/deckhouse/pull/1819#discussion_r901856171

Reply:
I have a strong opinion that every script executed on CI can be executed locally as well (and should produce the same result).

If we only expect this script to run on CI, why is gh tool used for API interactions to open prs instead? Why do we check dependencies before the start?

Reply:
Okay, I'm sure @RomanenkoDenys definitely achieved that locally. 

IMO, it's enough to automate everything except PR creation to say that we have the same tool for both CI and developer machine. Running `make` and getting the diff will do.

GA lets us not bother with such a dependency as `gh`. Some developers don't use `gh` at all (I do). Some developers would prefer to open web interface and type there.

I would prefer not to bother developers at all with such a routine. And make it fully on daily basis acutomatically. The workflow is as simple as running `make` and creating a PR like we do it for changelogs. This is easy to achieve.

Reply:
Definitely agree! Had the same though.

Reply:
i removed pr creation, we will add GA later